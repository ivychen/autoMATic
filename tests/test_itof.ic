int main() {
	int i = 69;
	float f = itof(i);
	print(f);
	print(itof(8008135));
	return 0;
}
