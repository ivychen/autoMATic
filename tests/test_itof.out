69.000000
8008135.000000
